src/app/content/highlights/components/CardWrapper.tsx

```diff
@@ -97,9 +122,14 @@ const Wrapper = ({highlights, className, container, highlighter}: WrapperProps)
     if (!position) { return; }
     const topOffset = getOffsetsForHighlight(focusedHighlight).top;
 
+    let transform = '';
```

Comment:
part of me just hates `let` and would prefer this be done in a ternary assignment, but i think thats just personal preference on my part

Reply:
np :D